app/views/react.scala.html

```diff
@@ -1,8 +1,16 @@
 @import assets.AssetsResolver
-@(title: String, id: String, js: String, description: Option[String] = None)(implicit assets: AssetsResolver, request: RequestHeader)
+@(
+    title: String,
+    id: String,
+    js: String,
+    description: Option[String] = None
+)(implicit assets: AssetsResolver, request: RequestHeader)
 
 @scripts = {
     <script type="text/javascript" src="@assets(js)"></script>
+    <script type="text/javascript">
+        window.guardian = window.guardian || {};
```

Comment:
Do you still need the guardian object in this template?

Reply:
ah, nope!